src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -114,6 +114,36 @@
     short-summary: Delete an SSL certificate from a web app.
 """
 
+helps['appservice web config ssl'] = """
+    type: group
+    short-summary: commands to configure ssl certs
+"""
+
+helps['appservice web config ssl upload'] = """
+    type: command
+    short-summary: Upload an ssl certificate to the webapp
+"""
+
+helps['appservice web config ssl list'] = """
+    type: command
+    short-summary: List ssl certificates within a resource group
+"""
+
+helps['appservice web config ssl bind'] = """
+    type: command
+    short-summary: Bind an ssl certificate to the webapp
+"""
+
+helps['appservice web config ssl unbind'] = """
+    type: command
+    short-summary: Unbind an ssl certificate from the webapp
+"""
+
+helps['appservice web config ssl delete'] = """
+    type: command
+    short-summary: Delete an ssl certificate from the webapp
+"""
```

Comment:
Looks like these values already exist in the file.
Modify those instead of duplicating.